frontend/e-buddy/src/app/(demo)/bots/setup/page.tsx

```
@@ -0,0 +1,159 @@
+                  <input 
+                    type="url" 
+                    placeholder="Enter URL to scrape" 
+                    className="w-full border border-gray-300 rounded-lg px-4 py-3 shadow-sm focus:outline-none focus:ring-2 focus:ring-blue-500 transition"
+                  />
+                </div>
+              </form>
+              <div className="mt-8 flex justify-between">
+                <button 
+                  onClick={prevStep} 
+                  className="bg-gray-300 text-gray-700 px-6 py-3 rounded-lg hover:bg-gray-400 transition-colors"
+                >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)